tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/debugger_container_test.ts

```diff
@@ -341,8 +342,93 @@ describe('Debugger Container', () => {
       expect(digestsWithAlert[0].nativeElement.innerText).toEqual('6');
       expect(digestsWithAlert[1].nativeElement.innerText).toEqual('8');
     });
+
+    for (const numExecutions of [1, 9, 10]) {
+      it(`hides slider if # of executions ${numExecutions} <= display count`, () => {
+        const fixture = TestBed.createComponent(TimelineContainer);
+        fixture.detectChanges();
+        const scrollBeginIndex = 0;
+        const displayCount = 10;
+        const opTypes: string[] = new Array<string>(numExecutions);
+        opTypes.fill('MatMul');
+        const debuggerState = createDebuggerStateWithLoadedExecutionDigests(
+          scrollBeginIndex,
+          displayCount,
+          opTypes
+        );
+        store.setState(createState(debuggerState));
+        fixture.detectChanges();
+
+        const sliders = fixture.debugElement.queryAll(
+          By.css('.timeline-slider')
+        );
+        expect(sliders.length).toBe(0);
+      });
+    }
+
+    for (const numExecutions of [11, 12, 20]) {
+      const displayCount = 10;
+      for (const scrollBeginIndex of [0, 1, numExecutions - displayCount]) {
+        it(
+          `shows slider if # of executions ${numExecutions} > display count,` +
```

Comment:
need a whitespace at the end the spec read better.

Reply:
Done.